landing-2/signin.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>로그인</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/sign.css" />
+  </head>
+  <body>
+    <header>
+      <a class="logo-link" href="index.html">
+        <img class="header-logo" src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+      </a>
+      <p class="header-message">
+        회원이 아니신가요?<a class="header-link" href="signup.html">회원 가입하기</a>
```

Comment:
상대경로로 href를 수정해주면 더욱 좋을 것 같습니다!
```suggestion
        회원이 아니신가요?<a class="header-link" href="./signup.html">회원 가입하기</a>
```
상대경로를 사용하는 이유 : 파일 시스템에서만 작동하는 환경이라면 상대 경로가 더욱 중요합니다. 이러한 방식으로 상대 경로를 사용하면, 파일 구조가 변경되더라도 링크가 올바르게 작동할 가능성이 높아집니다.

하지만 나중에는 번들러라는 친구에 의해 상대경로를 절대경로로 바꿀 수 있는데요 이부분은 리액트에서 번들러 라는 친구를 언급 하면서 한번 더 말씀 드리도록 하겠습니다. 지금은 아~ 번들러라는 키워드가 있구나~ 라는 것만 알고 가시면 좋을 것 같습니다!